Bukkit/src/main/java/com/plotsquared/bukkit/util/BukkitInventoryUtil.java

```diff
@@ -104,14 +109,19 @@ public void close(PlotInventory inv) {
     }
 
     @Override
-    public void setItem(PlotInventory inv, int index, PlotItemStack item) {
+    public boolean setItem(PlotInventory inv, int index, PlotItemStack item) {
```

Comment:
Is it not a breaking change as the signature of the method changing will make it unable to be compiled against for previous versions?

Reply:
Compiling would be fine but this is a binary incompatible change.

Reply:
Yeah I worded that really poorly. We should make a new method that returns boolean setItemAndReturn or setItemAndCheck or something and deprecate the old one (or leave and add javadoc note on the super method regarding this